signac/core/json.py

```diff
@@ -12,26 +15,7 @@
 except ImportError:
     NUMPY = False
 
-try:
-    import rapidjson as json
-    from rapidjson import Encoder
-    try:
-        # Defined for rapidjson >= 0.7.1
-        from rapidjson import JSONDecodeError
-    except ImportError:
-        # rapidjson < 0.7.1 raises a ValueError
-        JSONDecodeError = ValueError
-
-    class JSONEncoder(Encoder):
-        encode = Encoder.__call__
-
-    logger.debug(msg.format('rapidjson'))
-except ImportError:
-    import json
-    from json import JSONEncoder
-    from json.decoder import JSONDecodeError
-
-    logger.debug(msg.format('json'))
+logger.debug(msg.format('json'))
 
 
 _has_default = hasattr(JSONEncoder, 'default')
```

Comment:
This seems like a compatibility feature related to rapidjson, I am not sure if it is still needed with the plain `json` module. See if you can verify this in the documentation for JSONEncoder.